rclrs/src/node/mod.rs

```diff
@@ -212,4 +212,38 @@ impl Node {
             .filter_map(Weak::upgrade)
             .collect()
     }
+
+    /// Returns the ROS domain ID that the node is using.
+    ///    
+    /// returned values depend on ROS_DOMAIN_ID environment variable.   
+    ///
+    /// # Example
+    /// ```
+    /// use rclrs::{Context, RclReturnCode};
+    ///
+    /// // set default ROS domain ID to 10 here
+    /// std::env::set_var("ROS_DOMAIN_ID", "10");
+    /// let context = Context::new(std::vec![])?;
+    /// let node = context.create_node("domain_id_node")?;
+    /// let domain_id = node.get_domain_id()?;
+    ///
+    /// assert_eq!(domain_id, 10);
+    /// # Ok::<(), RclReturnCode>(())
+    /// ```
+    ///
+    /// TODO: If node option is supported,
+    /// add description about this function is for getting actual domain_id
+    /// and about override of domain_id via node option
+    pub fn get_domain_id(&self) -> Result<usize, RclReturnCode> {
```

Comment:
I think this might actually be a candidate for a function that just returns `usize`, see https://github.com/ros2-rust/ros2_rust/blob/doc_articles/docs/Contributing.md#wrapping-rcl-functions

Reply:
You are right. I thought RCL_RET_ERROR may be returned here before, but I realized that there is no chance by reading rcl source code, so fix to return `usize` merely.